README.md

```diff
@@ -366,15 +366,16 @@ boolean CheckOptInValue()
 
 - `OptInParams()` will only send device-related parameters that are specified. `OptOutParams()` will send all device-related parameters except ones that are specified.
 
-- Please note that we require the following parameters to properly track devices in Tenjin's system:
-
-  - `ip_address`
-  - `advertising_id`
-  - `limit_ad_tracking`
-  - `referrer` (Android)
-  - `iad` (iOS)
-
-- If you are targeting IMEI and/or OAID Ad Networks, add:
+- Please note that we require the following parameters to properly track devices in Tenjin's system. If one of these mandatory parameters is missing the event will not be processed or recorded.
+
+  - For Android, 
+    - `ip_address`
```

Comment:
Does IP address needs to be listed as a parameter? It's inherent in the requests we receive.

Reply:
thanks, updated